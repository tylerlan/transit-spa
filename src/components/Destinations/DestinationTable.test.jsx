import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { DestinationTable } from './DestinationTable';

describe('DestinationTable', () => {
  it('should render an empty table', () => {
    const component = shallow(<DestinationTable />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
