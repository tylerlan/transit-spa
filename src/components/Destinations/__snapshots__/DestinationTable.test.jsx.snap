// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DestinationTable should render an empty table 1`] = `
<div>
  <Segment>
    <DestinationRow
      id={1}
      name=""
    />
  </Segment>
</div>
`;
